### DISCUSSION TITLE
Uncaught Error: Class 'Phalcon\Mvc\Collection' not found

### DISCUSSION THREAD

**Discussion by ZMNOTZ** [phalcon/cphalcon#15763] [Q&A]
phalcon version:4.1.2
incubator version:3.4.6
php version: 7.3.29
I'm using ODM then I got the error.

**Comment by niden** [comment]
The MongoDB functionality has been removed from v4. We had to update the underlying driver to connect to Mongo and that brought the issue that one would have to have the mongo extension loaded before Phalcon whether it was to be used or not.

The whole package will be reintroduced in v6 since it will be easier for people to include it in their projects when needed.

In the meantime you can check this repository for the functionality: https://github.com/phalcon/incubator-mongodb

**Comment by ZMNOTZ** [comment]
I'm using incubator 3.4.6!!!





------------------ Original ------------------
From: Nikolaos Dimopoulos ***@***.***&gt;
Date: Fri,Nov 5,2021 0:52 AM
To: phalcon/cphalcon ***@***.***&gt;
Cc: Yohoho ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [phalcon/cphalcon] Uncaught Error: Class 'Phalcon\Mvc\Collection'not found (Discussion #15763)





 
The MongoDB functionality has been removed from v4. We had to update the underlying driver to connect to Mongo and that brought the issue that one would have to have the mongo extension loaded before Phalcon whether it was to be used or not.
 
The whole package will be reintroduced in v6 since it will be easier for people to include it in their projects when needed.
 
In the meantime you can check this repository for the functionality: https://github.com/phalcon/incubator-mongodb
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
